shared/components/map.js

```diff
@@ -11,6 +11,7 @@ import eventClickPng from '../resources/marker/event_big.png';
 import facilityClickPng from '../resources/marker/facility_big.png';
 import warningClickPng from '../resources/marker/warning_big.png';
 import {API_GEODATA, API_KEY} from '../utils';
+import LocationServicesDialogBox from 'react-native-android-location-services-dialog-box';

```

Comment:
we do not need another module for IOS since IOS have default location service dialog box.